.github/workflows/requirements.txt

```diff
@@ -8,3 +8,4 @@ pymongo==3.10.1
 mongomock==3.19.0
 django==2.2.10
 elasticsearch-dsl==6.4.0
+Jinja2==2.11.1
```

Comment:
Do you want to add this to the eager requirements as well? Maybe just setting `Jinja2<3.0.0` or something?